CHANGELOG.md

```diff
@@ -1,3 +1,149 @@
+# v1.1.0 (2021-05-07)
```

Comment:
Note: this PR assumes the following PRs will be merged:
* #1547 
* #1558 
* #1560

We no longer expect #1549 to be in this release.